[FCE-920] Describe metadata (#63)
fishjam-cloud/documentation: docs/_common/metadata/reading.mdx

## Reading metadata

Peer metadata are available as `metadata` property for each peer. When you list peers with `usePeers` hook, you can also read

> metadata associated with it. Note that `metadata.peer` property contains metadata set by client SDK (as in above examples).

Reviewer: **nitpick (blocking):** 
```suggestion
metadata associated with it. Note that `metadata.peer` property contains metadata set by the client SDK (as in examples above).
```

> metadata associated with it. Note that `metadata.peer` property contains metadata set by client SDK (as in above examples).
> And `metadata.server` data are set with server SDK as [described here](/production/server#metadata).

Reviewer: **nitpick (blocking):** 
```suggestion
metadata associated with it. Note that `metadata.peer` property contains metadata set by client SDK (as in above examples) and `metadata.server` data is set with the server SDK as [described here](/production/server#metadata).
```

> metadata associated with it. Note that `metadata.peer` property contains metadata set by client SDK (as in above examples).
> And `metadata.server` data are set with server SDK as [described here](/production/server#metadata).
> 

Reviewer 2: **suggestion:** 
```suggestion
metadata associated with it.
Note that `metadata.peer` property contains metadata set by client SDK (as in above examples).
The metadata set on the server side are available at `metadata.server`.
Learn more about server metadata [here](/production/server#metadata).

```
